test/unit/lodash.spec.js

```diff
@@ -0,0 +1,628 @@
+                var fun = _.over.apply(_, params);
+                expect(fun(3, 4, 2, 8, 7)).to.eql([8,2]);
+                expect(fun(-3, 0, 2, 88, 32, 12)).to.eql([88, -3]);
+            });
+        });
+    });
+
+    describe('overEvery', function() {
+        describe('overEvery1', function() {
+            var params,
+                elem1,
+                elem2;
+            before(function() {
+                params = datasets.overEvery1();
+                elem1 = params[0];
+                elem2 = params[1];
+            });
+            it('should check type', function() {
+                expect(elem1 instanceof Function).to.eql(true);
+                expect(elem2 instanceof Function).to.eql(true);
+                expect(params).to.have.length(2);
+            });
+            it('should return boolean from given class or functions', function(){
+                var fun = _.overEvery.apply(_, params);
+                expect(fun(5)).to.eql(true);
+                expect(fun(NaN)).to.eql(false);
+            });
+        });
+        
+        describe('overEvery2', function() {
+            var params,
+                elem1;
+            before(function() {
+                params = datasets.overEvery2();
+                elem1 = params[0];
+            });
+            it('should check type', function() {
+                expect(elem1 instanceof Function).to.eql(true);
+            });
+            it('should return boolean from given class', function(){
+                var fun = _.overEvery.apply(_, params);
+                expect(fun('lodash')).to.eql(true);
+                expect(fun(undefined)).to.eql(false);
+            });
+        });
+
+        describe('overEvery3', function() {
+            var params,
+                elem1;
+            before(function() {
+                params = datasets.overEvery3();
+                elem1 = params[0];
+            });
+            it('should check type', function() {
+                //expect(elem1 instanceof Array).to.eql(true);
+                expect(elem1 instanceof Function).to.eql(true);
+                expect(elem1 instanceof Function).to.eql(true);
+            });
+            it('should return boolean from given predicate', function(){
+                var fun = _.overEvery.apply(_, params);
+                expect(fun(7)).to.eql(true);
+                expect(fun('4')).to.eql(true);
+                expect(fun('five')).to.eql(false);
+                expect(fun(null)).to.eql(false);
+            });
+        });
+    });
+    describe('overSome', function() {
+        describe('overSome1', function() {
+            var params,
+                elem1,
+                elem2;
+            before(function() {
+                params = datasets.overSome1();
+                elem1 = params[0];
+                elem2 = params[1];
+            });
+            it('should check type', function() {
+                expect(elem1 instanceof Function).to.eql(true);
```

Comment:
You should check also false value. Now user can pass function which always return true.